### PR TITLE
Task 4 ofir review

### PR DESCRIPTION
**Pull request by Galcarmi**

### CODE REVIEW

File: express-server/index.js [hunk 1]
```diff
@@ -1,44 +1,55 @@
...
-  res.sendFile(`${eClientLocations.TASK_4_CLIENT_DIST}/index.html`, {
+  res.sendFile(`${eClientLocations.PRODUCTION}/index.html`, {
     root: parentFolder,
   });
 }));
 
-app.post('/todo', wrapError((req, res) => {
-    const addedTodo = todoInMemoryDBManager.addTodo(req.userId, req.body);
+app.post('/todo', wrapError(async (req, res) => {
+    const addedTodo = await todoMongoDBManager.addTodo(req.userId, req.body);
```
Comment (ofirdagan) [inline express-server/index.js:37]: what happened that u needed to change everything to async?

File: todo-express-client/src/mvvc/ViewController.js [hunk 2]
```diff
@@ -32,23 +32,23 @@ export class ViewCtrl {
     renderTodoPage();
   }
 
-  _addTodo({ content, id, isDone }) {
+  _addTodo({ content, _id, isDone }) {
```
Comment (ofirdagan) [inline todo-express-client/src/mvvc/ViewController.js:35]: why id with`_`?

Comment (Galcarmi) [reply todo-express-client/src/mvvc/ViewController.js:35]: i wanted to be consistent over the client & server variable names, the server returns todos with _id property

File: express-server/index.js (hunk 1 above)
Comment (Galcarmi) [reply express-server/index.js:37]: todoMongoDBManager has ajax calls in its methods so i had to change the anonymous function to be async

**Pull request merged**